### ISSUE TITLE
Milestone-1: Creating the Introduction&Project Description Part

### ISSUE THREAD

**Issue by bahricanyesil** [bounswe/bounswe2022group2#126]
### Issue Description

We went over the first milestone assignment description pdf in our regular meeting of week 6. We decided the sub-titles/sub-tasks that should be included in the milestone report as to be compatible with the ones indicated in the pdf.

- This is a sub-issue of the main ["Documenting Milestone Report 1" issue](https://github.com/bounswe/bounswe2022group2/issues/128).
- There is a main title at the very first part of the milestone report, which is named as "Executive Summary". We have determined separate subtitles under this main title and one of them is "Introduction / Project Description".
- I will be writing the contents of this sub-section and will be sharing the written texts as I progress under this issue as comments, you can track my progress.

### Step Details

Steps that will be performed:

- [x] Go over the project description pdf, our requirements and other related wiki pages.
- [x] Write the content of Introduction-Project Description part.
- [ ] Receive review from @onurkomurcu and update the content accordingly.

### Final Actions

After I wrote and shared the texts, reviewer (@onurkomurcu) will be reviewing them and I will be doing necessary updates accordingly.

Finally, after all steps including the review are finalized, I will add the final version to the milestone report.

### Deadline of the Issue

13.03.2022 - Wednesday - 17:00

### Reviewer

Onur Kömürcü

### Deadline for the Review

14.03.2022 - Thursday - 20:00

**Comment by bahricanyesil** [comment]
## Info

I wrote the first paragraph of the introduction and wanted to share it without waiting to finalize the whole content of the introduction / project description part. It is just the first paragraph as I indicated and there will be more paragraphs after this one. Here's what I wrote:

Our project is an online learning platform project that will bring a new perspective to the education system. It stands out from the classical methodologies and some newly developed platforms with its flexible and simple yet comprehensive system. More importantly, its attitude on education constitutes its main layout and difference. Our learning platform offers an education system that is independent of the user titles, locations, ages, and so on. Everyone can be a learner about a topic s/he is interested in and can teach about other topics that s/he knows well from anywhere at any time!

**Comment by bahricanyesil** [comment]
## Info

- I wrote the whole introduction/project description section. It is now finalized and ready to be reviewed by @onurkomurcu. Here's the text:

Our project is an online learning platform project that will bring a new perspective to the education system. It stands out from the classical methodologies and some newly developed platforms with its flexible and simple yet comprehensive system. More importantly, its attitude on education constitutes its main layout and difference. Our learning platform offers an education system that is independent of the user titles, locations, ages, and so on. Everyone can be a learner about a topic s/he is interested in and can teach about other topics that s/he knows well from anywhere at any time!

One of the core values of our project is attaching importance to “learning” rather than “grading” contrary to what is done in the classical education systems. Learners will be able to progress their learning process by getting in touch with the lecturers or other learners. All learners will be able to use registered lesson resources, solve problems, take self-evaluating quizzes, ask/answer questions, discuss ideas, and so on. In short, it will not only be an individual learning experience. The project also enables users to communicate with other users to optimize the learning process. As a feature of this collaborative learning process, users will be able to organize in-person events/meetings and configure the details related to the events (such as geolocations, date, duration, capacity, and so on) inside the app.

Apart from these core features, our platform also contains some modern technologies and mechanisms. Semantic search, recommendations, browsing, annotations (by following W3 Web Annotation Model), taking&sharing&connecting notes, and hyperlinks are some of these components.

At the end of the learning process, learners will be able to evaluate the courses and give feedback via lecture pools to share their experience about a course/lecturer with the other learners. However, they are also able to evaluate and track themselves by solving problems, taking quizzes, and monitoring their progress in their profile with their accumulated learning experiences.

In summary, we're building the future education system of the modern world by also including some new aspects that will be more common in the next few years. We understand well the core issue of the classical educational methodologies, and institutions since we also experience the same problems. We are proposing suitable solutions to handle these problems and have a better education process.

**Comment by bahricanyesil** [comment]
## Info

I added this part to our [milestone report](https://docs.google.com/document/d/1UD8-7QPV0NNDUppmNQNW438jxSsc8MrYwiKMgWrXKsY/edit). I will update the content there if there will occur any changes here.

**Comment by onurkomurcu** [comment]
I have reviewed your work. Introduction part of the milestone is crucial to take attention of the reader. Therefore, it should be written striking as you write. Thank you for your excellent work @bahricanyesil !
